addons/sys_io/fnc_server.sqf

```diff
@@ -49,6 +49,7 @@ DFUNC(connectionFnc) = {
                 } else {
                     hint "ACRE CONNECTED";
                 };
+                call FUNC(ts3ChannelCheck);
```

Comment:
Why not call it after `GVAR(serverStarted)` is set to `true` on L55? Then you don't need that extra check in the function itself.